### ISSUE TITLE
Any way to force the game full screen on lower resolutions other than native?

### ISSUE THREAD

**Issue by rezatitan7** [sultim-t/xash-rt#144]
Although I have a workaround to get for example 1080p resolution on full-screen, which is changing my desktop resolution through display settings down to 1080p.

However, I like a way to 'just' keep the game on 1080p full-screen and when I alt tab outta the game, resolution changes back to my native one.

**Comment by 0x4E69676874466F78** [comment]
Use FSR.

**Comment by rezatitan7** [comment]
> Use FSR.

I use DLSS, but that doesn't affect the game to go full-screen.

**Comment by Nord1cWarr1or** [comment]
try to set `-fullscreen` in launch parameters

**Comment by rezatitan7** [comment]
> try to set `-fullscreen` in launch parameters

It still launches on window mode when the game is set to 1080p.

**Comment by mcbpete** [comment]
Are you running at quite a high resolution (e.g. at 4k ?) if so then this worked for me: https://github.com/sultim-t/xash-rt/issues/54#issuecomment-1493033699

**Comment by OctopusButtons** [comment]
Yeah it's strange that there's no Fullscreen / Windowed option on the graphics options screen.  I'm having to repeatedly use mouse to click on the MAXIMIZE window bar control which still leaves me with a white bar at most resolutions.
